wgpu-hal/Cargo.toml

```diff
@@ -132,20 +132,20 @@ version = "22.0.0"
 ash = { version = "0.38.0", optional = true }
 gpu-alloc = { version = "0.6", optional = true }
 gpu-descriptor = { version = "0.3", optional = true }
-smallvec = { version = "1", optional = true, features = ["union"] }
+smallvec = { workspace = true, optional = true, features = ["union"] }
 
-khronos-egl = { version = "6", features = ["dynamic"], optional = true }
-libloading = { version = ">=0.7, <0.9", optional = true }
-renderdoc-sys = { version = "1.1.0", optional = true }
+khronos-egl = { workspace = true, features = ["dynamic"], optional = true }
+libloading = { workspace = true, optional = true }
+renderdoc-sys = { workspace = true, optional = true }
 
 [target.'cfg(target_os = "emscripten")'.dependencies]
-khronos-egl = { version = "6", features = ["static", "no-pkg-config"] }
+khronos-egl = { workspace = true, features = ["static", "no-pkg-config"] }
 #Note: it's unused by emscripten, but we keep it to have single code base in egl.rs
-libloading = { version = ">=0.7, <0.9", optional = true }
+libloading = { workspace = true, optional = true }
 
 [target.'cfg(windows)'.dependencies]
 # backend: Dx12
-bit-set = { version = "0.8", optional = true }
+bit-set = { workspace = true, optional = true }
 range-alloc = { version = "0.1", optional = true }
 gpu-allocator = { version = "0.27", default-features = false, features = [
```

Comment:
These and everything below (outside of commentable diff range) are in the workspace table as well; any reason to _not_ inherit them?

Reply:
gpu-allocator is v0.26 in the workspace and I wanted to keep this as straight changes so far. (Nothing uses the v0.26 so there's no issue in updating it.)

I plan to do more, either as part of this PR or followups, but I wanted to make sure that there was no issue with me doing this to avoid potentially wasted effort.

Reply:
Just like you did for `bitflags`, let's bring the version in the workspace table in sync since this is the only user?

Reply:
Yeah, let's make merge the 0.27 over to `Cargo.toml`. 👍🏻